data/coalition outfits.txt

```diff
@@ -221,6 +221,28 @@ outfit "Large Steering Module"
 	"turning heat" 8.2
 	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships, but gains a considerable amount of flexibility."
 
+outfit "Enforcer Confrontation Gear"
+	category "Hand to Hand"
+	licenses
+		Heliarch
+	cost 230000
+	thumbnail "outfit/enforcer confrontation gear"
```

Comment:
With this description, I'd be more interested in having the image be the one that Becca made. The staff looks great, though.